docs/content/docs/usage/intl.md

```diff
@@ -190,30 +213,35 @@ Once you can have your translated content, we have to add two buttons in the hea
 
 ```javascript
 ...
+import {browserHistory} from "phenomic/lib/client"
 import {setLocale} from "actions/intl"
 ...
 class Header extends Component {
-  constructor() {
-    super()
-    this.updateLocale = this.updateLocale.bind(this)
-  }
-
-  updateLocale(locale) {
-    store.dispatch(setLocale(locale))
-  }
-
+  ...
   render() {
+    const {updateLocale} = this.props
     return (
       <header>
         <nav>
           ...
-          <div onClick={() => this.updateLocale("en")}>{"en"}</div>
-          <div onClick={() => this.updateLocale("fr")}>{"fr"}</div>
+          <div onClick={() => updateLocale("en")}>{"en"}</div>
+          <div onClick={() => updateLocale("fr")}>{"fr"}</div>
         </nav>
       </header>
     )
   }
 }
+
+export default connect(
+  null,
+  dispatch => ({
+    ...
+    updateLocale: (locale) => {
+      dispatch(setLocale(locale))
+      browserHistory.push("/")
```

Comment:
I think it's preferable to do this, to avoid potential flash with different languages (eg: full page refresh will give you that (bad) UX). Thoughts?